examples/systemToSystemXcmForeignAssets.ts

```diff
@@ -0,0 +1,56 @@
+/**
```

Comment:
Collectives parachain does not have asset or foreignAsset pallets. This example only makes sense if sending DOT to Collectives.

Reply:
great catch! removing this example since we have one of sending the relay asset already.